__tests__/acceptance/eslint-with-todo-formatter.ts

```diff
@@ -0,0 +1,168 @@
+import execa from 'execa';
+import stripAnsi from 'strip-ansi';
+import { FakeProject } from '../__utils__/fake-project';
+import { readFile as readFixture } from '../__utils__/read-file-cached';
+
+describe('eslint with todo formatter', function () {
+  let project: FakeProject;
+
+  function runEslintWithFormatter(args: string[], options: execa.Options = {}) {
+    const defaultOptions = Object.assign(options);
+    defaultOptions.reject = false;
+    defaultOptions.cwd = options.cwd || project.baseDir;
+
+    return execa(
+      './node_modules/.bin/eslint',
+      ['src', '--format', require.resolve('../..')],
+      defaultOptions
+    );
+  }
+
+  beforeEach(() => {
+    project = FakeProject.getInstance();
+  });
+
+  afterEach(async () => {
+    // await project.dispose();
+  });
+
+  it('should not emit anything when there are no errors or warnings', async () => {
+    project.files = {
+      ...project.files,
+      src: {
+        'no-problems.js': readFixture('with-no-problems.js'),
```

Comment:
I thought about adding this to my repo. I'll probably do this too, but we should maybe think about extracting this to a separate repo for leverage.

Reply:
I'll keep this here for now to keep it simple but please let me know if you think we should extract to a separate repo.